integration_tests/test_math.py

```diff
@@ -125,6 +125,13 @@ def test_log1p():
 def test_expm1():
     assert expm1(1.0) - 1.71828182845904509 < eps
 
+def test_trunc():
+   i: f64
+   i = trunc(3.5)
```

Comment:
As suggested above, here the return type will be `i64` so change the declaration with `i: i64`.

Reply:
Thanks, I'll implement these corrections right away!